hacking/tagger/tag.py

```diff
@@ -41,7 +41,7 @@
 
 DEFAULT_ANSIBLE_CORE_CHECKOUT = ROOT.parent.joinpath("ansible")
 DEFAULT_REMOTE = "origin"
-DEFAULT_ACTIVE_BRANCHES: tuple[str, ...] = ("stable-2.14", "stable-2.15", "stable-2.16")
+DEFAULT_ACTIVE_BRANCHES: tuple[str, ...] = ("stable-2.15", "stable-2.16", "stable-2.17")
```

Comment:
```suggestion
DEFAULT_ACTIVE_BRANCHES: tuple[str, ...] = ("stable-2.14", "stable-2.15", "stable-2.16", "stable-2.17")
```

Reply:
Thanks @gotmax23 you're right that 2.14 is still active until Ansible 10 goes out.

Reply:
We want to keep this here until ansible-core 2.14.z goes EOL. The tags should be created whether or not the corresponding community package version is still supported.